README.md

```diff
@@ -20,11 +20,9 @@ limitations under the License.
 
 # AMP HTML ⚡
 
-AMP HTML is a way to build web pages for static content that render with reliable, fast performance. It is our attempt at fixing what many perceive as painfully slow page load times – especially when reading content on the mobile web.
+AMP HTML is a way to build web pages for static content that render with reliable, fast performance. It is our attempt at fixing what many perceive as painfully slow page load times – especially when reading content on the mobile web. See also: [What Is AMP?](https://www.ampproject.org/docs/get_started/about-amp.html). 
```

Comment:
Maybe add the link to the "AMP HTML" text at the start instead of this sentence?  That'll make the two paragraphs flow a bit better